clang/test/OpenMP/declare_simd_ast_print.cpp

```diff
@@ -63,22 +63,22 @@ class VV {
   // CHECK-NEXT: int add(int a, int b) __attribute__((cold))    {
   // CHECK-NEXT: return a + b;
   // CHECK-NEXT: }
-  #pragma omp declare simd uniform(this, a) linear(val(b): a)
+  #pragma omp declare simd uniform(this, a) linear(b: a)
```

Comment:
Why do we need to remove val, ref, and uval clauses from these test cases? If they are not supported by 6.0 then please check them separately.